mod-clans/src/main/java/gg/solarmc/loader/clans/ClanManager.java

```diff
@@ -56,45 +53,59 @@
         return Optional.ofNullable(clans.getIfPresent(id));
     }
 
+
     /**
-     * Gets a clan from memory cache or from table if not present.
-     * @param transaction the tx
-     * @param id ID of clan requested
-     * @return the clan you asked for idiot
-     * @throws IllegalStateException if the clan isn't present in table
+     * Gets a clan by it's identifier
+     *
+     * This fetches from the cache.
+     * @param transaction the transaction
+     * @param id the clan's id
+     * @return the clan if present, empty if not.
      */
-    public Clan getClan(Transaction transaction, int id) {
-        return clans.get(id, i -> {
+    public Optional<Clan> getClanByID(Transaction transaction, int id) {
```

Comment:
```suggestion
    public Optional<Clan> getClanById(Transaction transaction, int id) {
```
I suggest we use Id consistently throughout our codebase. We already use Id in several places.

Reply:
? It says id

Reply:
What's your question?

Reply:
Earlier i was wondering what you meant by this but i didn't see the difference in capitalization (my mistake)